packages/core/src/components/overlay/overlay.tsx

```diff
@@ -324,7 +327,7 @@ export class Overlay extends React.Component<IOverlayProps, IOverlayState> {
         }
         if (enforceFocus) {
             // make sure document.activeElement is updated before bringing the focus back
-            requestAnimationFrame(() => this.bringFocusInsideOverlay());
+            this.bringFocusInsideOverlay();
```

Comment:
yes precisely, DRY! every time `bFIO` was called, it turned out we wanted to `rAF` it, so i changed in one place instead of five.